client/my-sites/site-settings/form-discussion.jsx

```diff
@@ -179,6 +182,13 @@ class SiteSettingsFormDiscussion extends Component {
 					onChange={ this.handleCommentOrder }>
 					<span>{ translate( 'Comments should be displayed with the older comments at the top of each page' ) }</span>
 				</CompactFormToggle>
+				<JetpackModuleToggle
+					disabled={ isRequestingSettings || isSavingSettings }
+					isJetpackSite={ isJetpack }
```

Comment:
You can safely remove `isJetpackSite={ isJetpack }` - it's already being added as a connected prop to `JetpackModuleToggle`.